pluginconfig/pluginconfig-311.yaml

```diff
@@ -50,6 +50,7 @@ versions:
       genevaTDAgent: osarpint.azurecr.io/acs/td-agent:master.20190228.1
       azureControllers: quay.io/openshift-on-azure/azure:v7.0
       canary: quay.io/openshift-on-azure/azure:v7.0
+      aroAdmissionController: placeholder
```

Comment:
can these placeholder lines be removed? if not, why not?

Reply:
Removed.